src/index.js

```diff
@@ -63,6 +64,10 @@ function IpfsAPI (host_or_multiaddr, port) {
       opts = {}
     }
 
+    if (typeof files === 'string' && files.startsWith('http')) {
+      files = request(files)
```

Comment:
Better add a test and docs for adding with a stream though, otherwise that might change in the future and then we don't realize what broke the add by url was actually the lack of add by stream